packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.0.1]
+
+### Added
+
+- Add missing `TransactionControllerOptions` type in package-level export ([#4683](https://github.com/MetaMask/core/pull/4683))
+
```

Comment:
Since we are effectively [extending the surface area of the package's API](https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md#additions), it can be categorized as minor: [Review new version strings](https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md#4-review-new-version-strings)